guidelines/enhancement_template.md

```diff
@@ -189,12 +193,17 @@ These are generalized examples to consider, in addition to the aforementioned [m
 - End user documentation, relative API stability
 - Sufficient test coverage
 - Gather feedback from users rather than just developers
+- Enumerate service level objectives (SLIs), expose SLIs as metrics
+- Write symptoms-based alerts for the component(s)
 
 ##### Tech Preview -> GA 
 
 - More testing (upgrade, downgrade, scale)
 - Sufficient time for feedback
 - Available by default
+- Backhaul SLI telemetry
```

Comment:
Can you clarify what exactly this means? either in a reply here if I seem dense or in the template if it seems like others may not be clear what this means.

Reply:
It means that every component should expose metrics that express its SLIs, and those SLIs should be shipped back to telemeter so that they can be measured, discussed, iterated upon and so forth, across the fleet.